src/main/java/com/dev/cinema/controller/InjectController.java

```diff
@@ -0,0 +1,37 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.Role;
+import com.dev.cinema.model.User;
+import com.dev.cinema.security.AuthenticationService;
+import com.dev.cinema.service.RoleService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/inject")
+public class InjectController {
+    private static final Logger LOGGER =
+            LogManager.getLogger(InjectController.class);
+    private final RoleService roleService;
+    private final AuthenticationService authenticationService;
+
+    public InjectController(RoleService roleService,
+                            AuthenticationService authenticationService) {
+        this.roleService = roleService;
+        this.authenticationService = authenticationService;
+    }
+
+    @GetMapping
```

Comment:
```suggestion
    @PostConstruct
```
спробуй так, тоді не потрібно буде ручками кидати запит на цей ендпойнт

Reply:
дякую, круто!